evap/staff/templates/staff_semester_view.html

```diff
@@ -13,30 +13,26 @@
         {{ semester.name }}
         {% if request.user.is_staff %}
             &nbsp;
-            <a href="{% url "staff:semester_edit" semester.id %}" class="btn btn-xs btn-default">{% trans "Edit" %}</a>
-            <a onclick="show_delete_semester_modal({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-xs btn-danger{% if not semester.can_staff_delete %} disabled{% endif %}">{% trans "Delete" %}</a>
-            <a onclick="show_archive_semester_modal({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-xs btn-danger{% if not semester.is_archiveable %} disabled{% endif %}">{% trans "Archive" %}</a>
+            <a href="{% url "staff:semester_edit" semester.id %}" class="btn btn-sm btn-secondary">{% trans "Edit" %}</a>
+            <a onclick="show_deleteSemesterModal({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-sm btn-danger{% if not semester.can_staff_delete %} disabled{% endif %}">{% trans "Delete" %}</a>
+            <a onclick="show_archiveSemesterModal({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-sm btn-warning{% if not semester.is_archiveable %} disabled{% endif %}">{% trans "Archive" %}</a>
         {% endif %}
     </h3>
 
     {% if num_courses > 0 %}
-    <div class="panel panel-info">
-        <div class="panel-heading">
-            <span class="panel-title">
-                <a class="accordion-toggle" data-toggle="collapse" href="#accordion"  onclick="saveCollapseState()">
-                    {% trans "Overview" %}
-                </a>
-            </span>
+    <div class="card mb-3">
+        <div class="card-header">
+            <a class="collapse-toggle" data-toggle="collapse" href="#overviewCard" aria-expanded="false" aria-controls="overviewCard" onclick="saveCollapseState()">{% trans "Overview" %}</a>
         </div>
-        <div class="panel-collapse in" id="accordion">
+        <div class="card-body collapse" id="overviewCard">
             <table class="table">
                 <thead>
                     <tr>
-                        <th class="col-sm-3">{% trans "Degree" %}</th>
-                        <th class="col-sm-2">{% trans "Evaluation period" %}</th>
-                        <th class="col-sm-2">{% trans "Evaluated courses" %}</th>
-                        <th class="col-sm-2">{% trans "Reviewed comments" %}</th>
-                        <th class="col-sm-3">{% trans "Participation" %}</th>
+                        <th style="width: 19%">{% trans "Degree" %}</th>
+                        <th style="width: 30%">{% trans "Evaluation period" %}</th>
+                        <th style="width: 17%">{% trans "Evaluated courses" %}</th>
+                        <th style="width: 17%">{% trans "Reviewed comments" %}</th>
+                        <th style="width: 17%">{% trans "Participation" %}</th>
```

Comment:
well actually, if you said when making evap more mobile friendly we have to use cols again, all these aren't a change for the better.

Reply:
see above